Tella/Data/Database/ServerDatabase.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  UwaziServerDatabase.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/14/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+// MARK: - Methods related to Uwazi Server
+extension TellaDataBase {
+    func alterTable() {
```

Comment:
I think `alterTable` is unuseful because we have uwazi server and tella web server separately